templates/partials/make-sure-you-leave.html

```diff
@@ -0,0 +1,9 @@
+{% from 'components/panel/_macro.njk' import onsPanel %}
```

Comment:
Not to sure on `make-sure-you-leave.html` as it seems a bit vague. Maybe it should be something like `sign-out-pre-footer-panel`?